<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fondo">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>691</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="fondo_menu">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>131</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#fondo_menu{
	background-color: rgb(37, 161, 148);
	border-radius: 20px;
}

menu_icon{
	background-color: rgb(255, 255, 255);
}

QVBoxLayout{
	background-color: transparent;
}

QPushButton{
	background-color: transparent;
	text-align: left;
	color: white;
}

QLabel{
	background-color: transparent;
	color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="icon_user">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../src/icons/icon_user.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="icon_unimayor">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../src/icons/logo_IUCMC.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_userName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UserName</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_userRole">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UserRole</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_button_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>239</y>
        <width>109</width>
        <height>48</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Teoría</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../src/icons/ico_home.png</normaloff>../src/icons/ico_home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_button_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>341</y>
        <width>109</width>
        <height>48</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Quiz</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../src/icons/ico_home.png</normaloff>../src/icons/ico_home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>188</y>
        <width>109</width>
        <height>48</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../src/icons/ico_home.png</normaloff>../src/icons/ico_home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="menu_button_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>109</width>
        <height>48</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Prácticas</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../src/icons/ico_home.png</normaloff>../src/icons/ico_home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>9</y>
       <width>531</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(248, 249, 250);</string>
        </property>
        <widget class="QWidget" name="page_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 249, 250);</string>
         </property>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>-1</y>
            <width>531</width>
            <height>591</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	background-color: rgb(248, 249, 250);
}

QLabel{
	background-color: transparent;
}

QPushButton{
	background-color: rgb(96, 189, 218);
	border-radius: 5px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_title_home">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Bienvenido</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_home_practicas">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>170</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Prácticas</string>
           </property>
          </widget>
          <widget class="QLabel" name="image_home_practicas">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>210</y>
             <width>121</width>
             <height>131</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../src/images/image_practic.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="button_home_practicas">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>380</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Ingresar</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame_2">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>160</y>
             <width>141</width>
             <height>261</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">	background-color: rgb(232, 232, 232);
	border-radius: 15px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
          <widget class="QFrame" name="frame_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>160</y>
             <width>141</width>
             <height>261</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">	background-color: rgb(232, 232, 232);
	border-radius: 15px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
          <widget class="QLabel" name="image_home_teoria">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>210</y>
             <width>121</width>
             <height>131</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../src/images/image_teoria.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="button_home_teoria">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>380</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Ingresar</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_home_teoria">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>170</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Teoría</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame_4">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>160</y>
             <width>141</width>
             <height>261</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">	background-color: rgb(232, 232, 232);
	border-radius: 15px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_home_quiz">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>10</y>
              <width>41</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Quiz</string>
            </property>
           </widget>
          </widget>
          <widget class="QLabel" name="image_home_quiz">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>210</y>
             <width>121</width>
             <height>131</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../src/images/image_quiz.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="button_home_quiz">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>380</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Ingresar</string>
           </property>
          </widget>
          <zorder>frame_2</zorder>
          <zorder>label_title_home</zorder>
          <zorder>label_home_practicas</zorder>
          <zorder>image_home_practicas</zorder>
          <zorder>button_home_practicas</zorder>
          <zorder>frame_3</zorder>
          <zorder>image_home_teoria</zorder>
          <zorder>button_home_teoria</zorder>
          <zorder>label_home_teoria</zorder>
          <zorder>frame_4</zorder>
          <zorder>image_home_quiz</zorder>
          <zorder>button_home_quiz</zorder>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
